testing/simulator/src/checks.rs

```diff
@@ -329,6 +334,20 @@ pub(crate) async fn verify_light_client_updates<E: EthSpec>(
                 "Existing finality update too old: signature slot {signature_slot}, current slot {slot:?}"
             ));
         }
+
+        let light_client_updates = client
+            .get_beacon_light_client_updates::<E>(sync_committee_period, 1)
+            .await
+            .map_err(|e| format!("Error while getting light client update: {:?}", e))?
+            .ok_or(format!("Light client update not found {slot:?}"))?;
+
+        // Ensure we're only storing a single light client update for the given sync committee period
+        if light_client_updates.len() != 1 {
+            return Err(format!(
+                "{} light client update(s) was returned when only one was expected.",
```

Comment:
```suggestion
                "{} light client updates were returned when only one was expected.",
```

Nitpick: we know the number isn't 1 so we can use the plural (0 is plural)